wagtail/users/templatetags/wagtailusers_tags.py

```diff
@@ -185,8 +187,24 @@ def format_permissions(permission_bound_field):
 
 @register.inclusion_tag("wagtailadmin/shared/buttons.html", takes_context=True)
 def user_listing_buttons(context, user):
-    button_hooks = hooks.get_hooks("register_user_listing_buttons")
-    buttons = sorted(
-        itertools.chain.from_iterable(hook(context, user) for hook in button_hooks)
+    warn(
+        "`user_listing_buttons` template tag is deprecated.",
+        category=RemovedInWagtail70Warning,
```

Comment:
In favour of the more-standardised approach of defining the buttons in the view code and rendering them as components inside the Column's cell template, like we do for the generic model views.